include/zephyr/bluetooth/uuid.h

```diff
@@ -5249,6 +5249,27 @@ bool bt_uuid_create(struct bt_uuid *uuid, const uint8_t *data, uint8_t data_len)
  */
 void bt_uuid_to_str(const struct bt_uuid *uuid, char *str, size_t len);
 
+/** @brief Convert string to Bluetooth UUID.
+ *
+ *  Converts string to Bluetooth UUID. The string must be in the standard
+ *  UUID format, which could be 16 bit, 32 bit or 128 bit format. And the
+ *  uuid pointer must point to a bt_uuid large enough to hold the converted
+ *  UUID. If the string uuid is 16 bit, the converted bt_uuid will be
+ *  bt_uuid_16 type. If the string uuid is 32 bt uuid, the converted bt_uuid
```

Comment:
```suggestion
 *  bt_uuid_16 type. If the string uuid is 32 bit uuid, the converted bt_uuid
```

or did you mean to use `BT`/Bluetooth?